spacelift/type_conversions.go

```diff
@@ -88,3 +88,19 @@ func toOptionalStringMap(input interface{}) *structs.StringMap {
 	}
 	return &customHeaders
 }
+
+// getOptionalBool checks if user has explicitly set a boolean value in the configuration and returns it as a pointer to graphql.Boolean.
+// If the value is not set in the configuration, it returns nil.
+func getOptionalBool(d *schema.ResourceData, key string) *graphql.Boolean {
+	// Check if the raw config has the key set explicitly
+	rawConfig := d.GetRawConfig()
+	if !rawConfig.IsNull() {
```

Comment:
Nit: I prefer to keep the code as close to the left as possible. I believe it makes the code path easier to read.
Could we invert the conditions and go with early returns?